makerdao/governance-portal-v2
# refactor delegations fetching

### What does this PR do?

- Fetches kovan delegates from https://github.com/makerdao-dux/voting-delegates
- Fetches kovan and mainnet delegates from serverside paths. Disabling clientside fetching and minimizing the number of requests.
- Remove last voted date 
- ⚠️ Introduces `getMaker(network)` that returns a instance of maker for the specified network. This is usefull to fetch server side kovan information
- Adds styles for tables for markdown rendering as suggested by @LongForWisdom
- Separate FScache into a module, add expiration date
- Fetch delegates through API requests on client side (with the cache included)

### Steps for testing:

1. Go to delegates page in kovan
2. See a recognized delegate and can access to their profile

1. See that http://localhost:3000/api/delegates?network=kovan works
2. See that http://localhost:3000/api/delegates/0xc8829647c8e4131a01354ccac993388568d12d00?network=kovan works
3. See that cache is being regenerated/dumped for delegates API every 30 seconds



### Screenshots (if relevant):
![image](https://user-images.githubusercontent.com/1152768/125667472-68b7f418-41cb-4fd3-8304-83526b9dc2b6.png)

![image](https://user-images.githubusercontent.com/1152768/125667502-c4d6e078-5dec-4a03-8264-e940b961851b.png)

![image](https://user-images.githubusercontent.com/1152768/125761883-7f8d18d2-8bc6-4a94-829d-699050b8a9de.png)
